### PR TITLE
feat: update account endpoint

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-id-mobile/issues/2 |
| Dependencies | https://github.com/ripe-tech/ripe-id/pull/16 |
| Decisions | Endpoint to edit the user account information. |
| Animated GIF | -- |

### CODE REVIEW

File: js/account.js [hunk 1]
```diff
@@ -6,6 +6,20 @@ export const AccountAPI = superclass =>
...
+        async updateSelfAccount(account) {
+            const url = this.baseUrl + "accounts/me";
+
+            const dataM = {};
+            if (account.avatar) dataM.avatar = account.avatar;
+            if (account.description) dataM.description = account.description;
+            if (account.meta_extra) dataM.meta_extra = JSON.stringify(account.meta_extra);
+
+            const contents = await this.put(url, {
+                dataM: dataM
```
Comment (gcandal) [inline js/account.js]: Is there any problem with doing `dataM: account`?

Comment (gcandal) [reply js/account.js]: I know it's different from ripe-sdk, but at the same time it's coherent with the rest of these yonius API clients.

Comment (BeeMargarida) [reply js/account.js]: I wanted to make it similar to ripe-sdk, and also the JSON.stringify part. Do I pass the account and override the meta_extra?

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Comment by joamag** [comment]
@BeeMargarida check the changes

**Pull request merged**